src/common/hooks/useFocused.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
```

Comment:
I dont see this being used anywhere. Am I missing it?

Reply:
No longer used will delete